extensions/wikia/DesignSystem/scripts/DesignSystemGlobalNavigationOnSiteNotifications.text-formatter.js

```diff
@@ -131,6 +133,12 @@ define('ext.wikia.design-system.on-site-notifications.text-formatter', [
 					'notifications-reply-upvote-multiple-users-with-title' :
 					'notifications-reply-upvote-multiple-users-no-title';
 			};
+
+			this._getTalkPageMessageText = function (notification) {
+				return fillArgs(getMessage('notifications-talk-page-message'), args = {
```

Comment:
Do you need the `args = ` ?

Reply:
That could explain why it's not working.